test/integration/narrow/app/fuzzySearchService.test.js

```diff
@@ -174,4 +175,36 @@ describe('fuzzySearchService tests with filterBy', () => {
     expect(filteredResultPage2.count).toBe(11)
     expect(filteredResultPage2.rows.length).toBe(5)
   })
+
+  test('GET /paymentdata filters by single amount', async () => {
+    const amounts = ['5000-9999']
+    const filteredResult = await getPaymentData({ ...searchCriteria, filterBy: { amounts } })
+
+    filteredResult.rows.forEach(x => {
+      const matchingSet = paymentestdata.find(td => td.payee_name === x.payee_name)
```

Comment:
I think this one is already discussed in 383 to include part postcode also for comparison right?

Reply:
Yup, will change it in the new tests.